travharv/templates/execution_report.ttl

```diff
@@ -0,0 +1,59 @@
+{#
+   This file contains the template for the execution report 
+   :param: execution_report 
+   :param: task_reports
+#}
+
+@prefix prov: <http://www.w3.org/ns/prov#>.
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix void: <http://rdfs.org/ns/void#>.
+@prefix schema: <http://schema.org/>.
+@prefix travharv: <http://example.org/travharv/ns/>.
+
+<urn:travharv:{{execution_report.id}}> 
+    a prov:Entity, sh:ValidationReport ;
+
+    prov:generatedAtTime "{{execution_report.last_mod}}" ; 
+    travharv:fromContext "{{execution_report.config_name}}" ;
```

Comment:
in general I think it is best rdfj2-template-practice for all `{{insertions}}` to be made "safe" by applying the `| xsd` filter on them

so change to 
`{{execution_report.config_name | xsd("string")}}`